src/scss/_main.scss

```diff
@@ -606,8 +606,13 @@
 ///
 /// @param {String} $layout-name - One of $o-grid-layouts
 @mixin _oGridModifiersForLayout($layout-name) {
-	@include _oGridHumanFriendlyKeywords($layout-name);
-	@include _oGridShuffleColumns($layout-name);
+	@if $o-grid-human-friendly-selectors == true {
+		@include _oGridHumanFriendlyKeywords($layout-name);
+	}
+
+	@if $o-grid-offset-selectors == true {
```

Comment:
Agreed, we should probably aim to stick with current naming schemes to prevent confusion